phpunit.xml.dist

```diff
@@ -9,8 +9,8 @@
 >
     <php>
         <ini name="error_reporting" value="-1" />
-        <env name="APP_ENV" value="test" />
-        <env name="SHELL_VERBOSITY" value="-1" />
+        <server name="APP_ENV" value="test" force="true" />
+        <server name="SHELL_VERBOSITY" value="-1" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
```

Comment:
add `<env name="SYMFONY_PHPUNIT_REMOVE" value=""/>`

Reply:
That will install 2 different versions of `symfony/yaml` in our case one in the phpunit vendor and one in the our vendor.

Reply:
havent got any error in the workshop code